src/lib/apputils/net-server.c

```diff
@@ -807,20 +799,19 @@ setup_udp_port_1(struct socksetup *data, struct sockaddr *addr, int pktinfo)
         if (sock == -1)
             return 1;
         setnbio(sock);
-
-        if (pktinfo) {
-            r = set_pktinfo(sock, addr->sa_family);
-            if (r) {
-                com_err(data->prog, r,
-                        _("Cannot request packet info for udp socket address "
-                          "%s port %d"), paddr(addr), port);
-                close(sock);
-                return 1;
-            }
+        r = set_pktinfo(sock, addr->sa_family);
+        if (r) {
+            com_err(data->prog, r,
+                    _("Cannot request packet info for udp socket address "
+                        "%s port %d"), paddr(addr), port);
+            krb5_klog_syslog(LOG_INFO, _("System does not support pktinfo "
```

Comment:
I'd make this a LOG_WARNING.  The continuation lines for the string are indented to the wrong open paren; this may format better if you drop to the next line for the string argument, so that the string fragments can be longer.  Also, two spaces after the period.